Dockerfile

```diff
@@ -106,6 +106,16 @@ ENV APP_ENV=dev
 USER root
 ENV COMPOSER_ALLOW_SUPERUSER=true
 
+RUN apk add --no-cache --virtual .build-deps $PHPIZE_DEPS && \
+    pecl install \
```

Comment:
I don't like compiling things for a dynamic language, but no alternative

Reply:
Did wonder if the compilation could be done separately and the result copied in to save app changes from re-running it.

Reply:
In theory the end result is just a `.so` file in the right place, which if the PHP (and OS?) version match should be movable across filesystems. Maintaining the code to move it around might not be worth it though if the primary use case of `dev` is overriding the code with a host folder mounted as a volume.

Reply:
Will ignore for now, can revisit if it does become annoying.